plugins/usbpro/WidgetDetectorThreadTest.cpp

```diff
@@ -337,6 +353,37 @@ void WidgetDetectorThreadTest::testDmxterWidget() {
 }
 
 
+/**
+ * Check that we can locate a DMXUSB widget.
+ */
+void WidgetDetectorThreadTest::testDMXUSBWidget() {
+  uint8_t serial_data[] = {0x78, 0x56, 0x34, 0x12};
+  uint8_t manufacturer_data[] = "\153\152DMXUSB";
```

Comment:
Ah, I remember what I couldn't get to work: Where do the 153 and 152 come from? What should they be for my device? (lines 360-362).

Reply:
Without looking at the code, that as octal is hex 0x6b 0x6a, which is the manufacturer ID for JPK, makers of the DMXKing:
http://rdm.openlighting.org/manufacturer/display?manufacturer=27243

Reply:
Aha an octal! I'll push a commit.